tests/rtm/test_rtm_client.py

```diff
@@ -17,6 +17,14 @@ def setUp(self):
     def tearDown(self):
         slack.RTMClient._callbacks = collections.defaultdict(list)
 
+    def test_run_on_returns_callback(self):
+        @slack.RTMClient.run_on(event="message")
+        def fn_used_elsewhere(**payload):
+            pass
+        
+        self.assertTrue(fn_used_elsewhere != None)
+        self.assertTrue(fn_used_elsewhere.__name__ == "fn_used_elsewhere")
```

Comment:
`self.assertEqual(fn_used_elsewhere.__name__, "fn_used_elsewhere")` would be better.